### PR TITLE
Add macOS support

### PR DESCRIPTION
**Pull request by iRath96**
Hello there,

you've got a great repository here – it would be a shame if it didn't run on the "world's most advanced operating system" macOS 😉 Fear no more, I'm here to correct this disgrace!

Cheers,
– Alex

### CODE REVIEW

File: SimpleImageIO/SimpleImageIO.csproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
     <TargetFramework>net5.0</TargetFramework>
     <!-- Each additional platform requires an entry here, once the native binaries have been added below -->
-    <RuntimeIdentifiers>win10-x64;linux-x64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win10-x64;linux-x64;osx.10.15-x64</RuntimeIdentifiers>
```
Comment (pgrit) [inline SimpleImageIO/SimpleImageIO.csproj:18]: Are the macOS 10.15 binaries incompatible with other OS versions? If not, it's probably best to use the more general "osx-64", which corresponds to any version >= macOS 10.12 Sierra

File: README.md [hunk 2]
```diff
@@ -34,7 +34,12 @@ That's it. Simply add a reference to `SimpleImageIO/SimpleImageIO.csproj` to you
 
 ## Building on other platforms
 
-In theory, the package works on any platform. However, the native dependencies have to be built for each. Currently, the workflow has been set up and tested for 64 bit Windows and Linux (Ubuntu 20.04). Other platforms (like OSX) need to be built from source. For these, the [SimpleImageIO.csproj](SimpleImageIO/SimpleImageIO.csproj) file needs to be adjusted, instructions can be found in the comments of that file. The process should be a simple copy&paste operation, provided nothing goes south when building the C++ library.
+In theory, the package works on any platform.
+However, the native dependencies have to be built for each.
+Currently, the workflow has been set up and tested for Intel 64bit Windows, Linux (Ubuntu 20.04) and macOS 10.15.
```
Comment (pgrit) [inline README.md:39]: Best rephrase to x86-64bit as it also runs on AMD ;)

Comment (iRath96) [reply README.md:39]: True, x86-64 is probably more commonly known (even though Intel 64 and AMD64 are often used as more readable synonyms for x86-64).

File: SimpleImageIO/SimpleImageIO.csproj (hunk 1 above)
Comment (iRath96) [reply SimpleImageIO/SimpleImageIO.csproj]: Fixed in fa174c4.

File: README.md (hunk 2 above)
Comment (iRath96) [reply README.md]: Fixed in 33b176e.

**Pull request merged**